api/songs/respository/song-repository.ts

```diff
@@ -231,15 +231,39 @@ export const createSongRepository = (client: DynamoDB) => {
       },
       UpdateExpression: "SET voteCount = :voteCount REMOVE GSI2PK, voters",
     });
-
     // throw "NYI: need to remove votes";
   };
 
+  const addLockToSong = async (id: string): Promise<void> => {
+    await client.updateItem({
+      TableName: "song",
+      Key: {
+        PK: {
+          S: id,
+        },
+        SK: {
+          S: id,
+        },
+      },
+      ReturnValues: "UPDATED_NEW",
+      ExpressionAttributeValues: {
+        ":zero": {
```

Comment:
The ":zero" definition can be taken away since you aren't using it for this case. It's not doing any harm, but it is a little confusing to have it there.

Reply:
This has been cleaned up.